README.md

```diff
@@ -18,6 +18,10 @@ Launch the Command Pallete **(Cmd+Shift+P / Ctrl+Shift+P)** and type `random` to
 
 Random values generation is also supported when using **multiple editors**.
 
+# Configuration
+
+* `vscodeRandom.contextMenu.enabled` : Enable/disable the context menu for the available data generators. Default is `false`.
```

Comment:
I wasn't sure about adding this feature to avoid having too many options in the context menu, but since we can enable it with a configuration, I think we can move forward with this 😄

Reply:
Also, can we change slightly this text to `...: Enable/disable the context menu for the available data generators (disabled by default).` ?

Reply:
Fixed